README.md

```diff
@@ -106,9 +106,9 @@ merged into the RFC repository as a markdown file. At that point the RFC is
   - Copy `0000-template.md` to `text/0000-my-feature.md` (where "my-feature" is
     descriptive. don't assign an RFC number yet).
   - Fill in the RFC. Put care into the details: RFCs that do not present
-    convincing motivation, demonstrate understanding of the impact of the
-    design, or are disingenuous about the drawbacks or alternatives tend to be
-    poorly-received.
+    convincing motivation, demonstrate lack of understanding of the impact
```

Comment:
Can you reword to "design's impact"? There are 3 "of" in a row now..

Reply:
Sure.